Documentation-Requirements.md

```diff
@@ -118,15 +118,15 @@ utility.
   in a *bash code block* with every command line prefixed with `$ ` to denote
   a shell prompt:
 
-  ```
+  <pre>
```

Comment:
I knew you'd pick this up :) I hear ya but there are no alternatives for this change afaics I'm afraid. Markdown parsers just cannot cope with embedded block in blocks so this was a relatively elegant solution to ensuring we can actually parse this document and thus validate it.

Reply:
Heh, maybe it is just github parsing that handles these....
four prepending spaces
```
    ```xyz```
```

four backticks around the outer block...
````
```xyz```
````

Reply:
"Four backticks"?

Reply:
yep, apparently you can nest `n` backticks inside `n+1` backtick sections... so, I will do a 5 backtick to try and show what I did.....

`````
````
```xyz```
````
`````

And, wow, it works! ;-)

Reply:
@grahamwhaley - github rendering isn't the problem - it's *parsing* the markdown that is ;)

However, there is a happy ending *I think* since I wasn't checking this doc locally using the latest version of the markdown parser. And using the latest, it now accepts the syntax!

Reply:
Gah - it's not the parsers that are the problem - it's *my* parser. I've got a modified [`kata-doc-to-script.sh`](https://github.com/kata-containers/tests/blob/master/.ci/kata-doc-to-script.sh) that this chokes on. I'll see if I can find another way to solve this issue by improving the parser somehow (may be a non-starter as that whole script is simple, by design - I'd rather not rewrite it in go unless we *really* need that).

Reply:
Sorry - can't do this for now (more trouble than it's worth, so I've reverted to using `<pre>` as that's a good quick solution I think (and isn't the only example of putting HTML in docs.